fastify.d.ts

```diff
@@ -71,28 +74,45 @@ export type FastifyServerOptions<
   RawServer extends RawServerBase = RawServerDefault,
   Logger = FastifyLoggerOptions<RawServer>
 > = {
+  http2?: boolean,
+  https?: https.ServerOptions
```

Comment:
These were specifically removed for the discriminant union overloading. Is there an issue with the types as is?

Reply:
Right, good catch. Fixing